### PR TITLE
Bug fix: Consider all PDF white-space characters in object header

### PR DESCRIPTION
**Pull request by LucianoHanna**

### CODE REVIEW

**Comment by k00ni** [comment]
@LucianoHanna thank you for your contribution!

Please add a test or two which demonstrate the problem you attempt to fix.

**Comment by LucianoHanna** [comment]
@k00ni I have added one test of what I attempted to fix.

File: src/Smalot/PdfParser/RawData/RawDataParser.php [hunk 1]
```diff
@@ -495,7 +495,9 @@ protected function getIndirectObject($pdfData, $xref, $objRef, $offset = 0, $dec
         $offset += strspn($pdfData, "\0\t\n\f\r ", $offset);
         // ignore leading zeros for object number
         $offset += strspn($pdfData, '0', $offset);
-        if (substr($pdfData, $offset, \strlen($objHeader)) !== $objHeader) {
+        // Newer pdf may use \n instead of whitespace
```
Comment (k00ni) [inline src/Smalot/PdfParser/RawData/RawDataParser.php:498]: What do you mean here with "Newer pdf"? Can you give me a reference to the specification?

Besides, could it be more than one \n at a time?

**Review by k00ni — CHANGES_REQUESTED** [review]
Thank for the tests. Almost done here, I just have one question.

File: src/Smalot/PdfParser/RawData/RawDataParser.php (hunk 1 above)
Comment (LucianoHanna) [reply src/Smalot/PdfParser/RawData/RawDataParser.php:498]: Maybe I mistook the use cases, I was comparing a file in PDF 1.4 that throws MissingCatalogException with a file in PDF 1.7 that works. Sorry about that.
I do not know the great details of the PDF specifications, but I believe that the reference in the specifications is in [table 1 (7.2.2)](https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf)

**Comment by LucianoHanna** [comment]
I made a new commit to consider all whitespace characters specified in this [document](https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf)

**Comment by k00ni** [comment]
Thank you @LucianoHanna

**Comment by LucianoHanna** [comment]
There's a release date for the next release?

**Comment by k00ni** [comment]
Nothing is planned but if there are no objections for #406 and #407 I can merge these next week and prepare a release.

**Pull request merged**